app/src/main/java/xyz/whereuat/whereuat/Contact.java

```diff
@@ -2,45 +2,29 @@
 
 import android.content.ContentValues;
 import android.content.Context;
-import android.util.Log;
 
-import xyz.whereuat.whereuat.db.DbObject;
-import xyz.whereuat.whereuat.db.DbTask;
 import xyz.whereuat.whereuat.db.command.InsertCommand;
+import xyz.whereuat.whereuat.db.command.QueryCommand;
 import xyz.whereuat.whereuat.db.entry.ContactEntry;
 
 /**
  * Created by kangp3 on 4/10/16.
  */
-public class Contact implements DbObject {
-    public final String name;
-    public final String phone;
-    public final boolean autoshare;
-
+public class Contact {
     private static final String TAG = "ContactObj";
 
-    public Contact(String _name, String _phone, boolean _autoshare) {
-        name = _name;
-        phone = _phone;
-        autoshare = _autoshare;
-    }
-
-    public void dbInsert(Context context) {
+    public static InsertCommand dbInsert(Context context, String name, String phone,
```

Comment:
Also, since these functions don't actually do inserts they just make the commands I feel like their names should reflect that, like `createInsertCommand` or something. `dbInsert` just seems misleading